src/main/java/com/playdata/eungae/member/controller/MemberApiController.java

```diff
@@ -24,25 +17,27 @@
 @RequestMapping("/api/my")
 public class MemberApiController {
 
-	private final MemberService memberService;
-
-	@PatchMapping("/profile/form/{memberSeq}")
-	public MemberUpdateResponseDto updateMemberInfo(@PathVariable Long memberSeq, 
-							@RequestBody MemberUpdateRequestDto updateRequestDto) {
-		return memberService.updateMemberInfo(memberSeq, updateRequestDto);
-	}
-
-	@PostMapping("/hospital")
-	@ResponseStatus(HttpStatus.CREATED)
-	public String appendFavorites(@RequestBody @Valid RequestFavoriesDto requestFavoriesDto) {
-		memberService.appendFavorites(requestFavoriesDto);
-		return "Favorites have been successfully appended";
-	}
-
-	@PatchMapping("/hospital")
-	@ResponseStatus(HttpStatus.ACCEPTED)
-	public String removeFavorites(@RequestBody @Valid RequestFavoriesDto requestFavoriesDto) {
-		memberService.removeFavorites(requestFavoriesDto);
-		return "Favorites have been successfully deleted";
-	}
+    private final MemberService memberService;
+
+    @ResponseStatus(HttpStatus.ACCEPTED)
```

Comment:
response code accepted 의 의미 입니다

_The HyperText Transfer Protocol (HTTP) 202 Accepted response status code indicates that the request has been accepted for processing, but the processing has not been completed; in fact, processing may not have started yet. The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place._

서버에서는 응답을 보냈지만 서버에서는 요청을 받아들였고 처리 되고 있음을 의미 합니다